### ISSUE TITLE
Optional Game Info Page

### ISSUE THREAD

**Issue by Gorian** [pterodactyl/panel#4699]
### Is there an existing feature request for this?

- [X] I have searched the existing issues before opening this feature request.

### Describe the feature you would like to see.

An optional "Information Page" - It would be great if I could have the front page / default landing page of pterodactyl be just information of hosted game servers along with auto-generated connection links.

### Describe the solution you'd like.

It would be nice to have an optional "info" page that you could choose as the panel's front page. You could, for example, generate a list of the hosted games, along with a [steam connection link](https://developer.valvesoftware.com/wiki/Steam_browser_protocol) that it autogenerated using the configured public ip/hostname, query port, and possible password. I.E. `steam://connect/{{Hostname}}:{{QueryPort}}/{{Password}}`. This could be a default field for eggs even. Optionally, it would be neat if it pulled info from the game, like the number of players, map that's playing, etc. Similar to these game banners:
![image](https://user-images.githubusercontent.com/1012176/219977874-f05e625d-689f-48a4-b581-6e0ea3a812f0.png)

**Comment by devnote-dev** [comment]
Be weary of https://github.com/pterodactyl/panel/issues/2717#issuecomment-728090469 ...

**Comment by Gorian** [comment]
> Be weary of https://github.com/pterodactyl/panel/issues/2717#issuecomment-728090469 ...

Hmmm... good point, but Steam Query is pretty game agnostic - many games support it.

**Comment by Boy132** [comment]
Duplicate of #1218

**Comment by Gorian** [comment]
Not really. There's a number of things that I mentioned that weren't mentioned in #1218, such as auto-generating `steam://` links. While it seems like they are asking for a server status page more for monitoring their servers, I'd like a front-page that you can share with users to make it easier to see what servers are hosted, and join them. Some potentially overlapping feature, but not necessarily a "duplicate"

**Comment by DaneEveritt** [comment]
In the interest of maintainer sanity and cleaning up an enormous backlog of stale feature requests we have moved to requiring all feature requests be discussed within [Discussions](https://github.com/orgs/pterodactyl/discussions/categories/feature-requests). Please consider re-opening this request there if you believe it is still relevant and worth consideration.

See https://github.com/pterodactyl/panel/issues/5491 for additional context.

Thanks.

<sub>_This comment is in the interest of issue tracker maintenance: it does not imply that maintainers are endorsing or dismissing your request._</sub>
